browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchFragment.java

```diff
@@ -36,12 +48,7 @@ public void onResume() {
      */
     @SuppressWarnings("WeakerAccess")
```

Comment:
do we still need to suppress this warning?

Reply:
Nope! Removing...